ocs_ci/helpers/helpers.py

```diff
@@ -2790,3 +2791,42 @@ def fetch_used_size(cbp_name, exp_val=None):
             f"matching. Retrying"
         )
     return used_in_gb
+
+
+def check_pvs_present_for_ocs_expansion(sc="localblock"):
+    """
+    Check for pvs present for OCS cluster expansion
+
+    Args:
+        sc (str): Name of SC
+
+
+    Returns:
+        bool: True if PVS present, assert if PVS not present
+
+    """
+    for rack_no in range(0, 3):
+        pv_check_list = list()
+        nodes_obj = OCP(
+            kind=constants.NODE,
+            selector=f"{constants.TOPOLOGY_ROOK_LABEL}=rack{rack_no}",
+        )
+        nodes_data = nodes_obj.get()["items"]
+        node_name = [nodes["metadata"]["name"] for nodes in nodes_data]
+        logger.info(node_name)
+        for nodes in node_name:
+            pvs_obj = OCP(
+                kind=constants.PV, selector=f"{constants.HOSTNAME_LABEL}={nodes}"
+            )
+            pvs_data = pvs_obj.get()["items"]
+            for pv in pvs_data:
+                if pv["spec"]["storageClassName"] == sc:
+                    if pv["status"]["phase"] == constants.STATUS_AVAILABLE:
+                        pv_check_list.append(pv["metadata"]["name"])
+        if pv_check_list:
+            pass
+        else:
+            raise PVNotSufficientException(
+                f"No Extra PV found in {constants.TOPOLOGY_ROOK_LABEL}=rack{rack_no} "
+            )
+    return True
```

Comment:
Since there is no condition in which this function can return False, I don't see a reason why to return True here.

Reply:
Done